src/sp_disabled_functions.c

```diff
@@ -391,43 +401,34 @@ bool should_disable(zend_execute_data* execute_data,
       sp_log_disable(complete_function_path, arg_name, arg_value_str,
                      config_node);
     }
-    if (true == config_node->simulation) {
-      goto next;
-    } else {  // We've got a match, the function won't be executed
-      return true;
-    }
+
   next:
     config = config->next;
   }
-allow:
-  return false;
 }
 
-bool should_drop_on_ret_ht(const zval* return_value, const char* function_name,
+void should_drop_on_ret_ht(const zval* return_value, const char* function_name,
                            const sp_list_node* config, const HashTable* ht,
                            zend_execute_data* execute_data) {
   const sp_list_node* ht_entry = NULL;
-  bool ret = false;
 
   if (!function_name) {
-    return ret;
+    return;
   }
 
   ht_entry = zend_hash_str_find_ptr(ht, function_name, strlen(function_name));
 
-  if (ht_entry &&
-      should_drop_on_ret(return_value, ht_entry, function_name, execute_data)) {
-    ret = true;
+  if (ht_entry) {
```

Comment:
Same comment [as previously](https://github.com/nbs-system/snuffleupagus/pull/255/files#r223580683), but I don't know if it's intended.